lua/obsidian/commands/init.lua

```diff
@@ -78,6 +50,25 @@ M.install = function(client)
 
     vim.api.nvim_create_user_command(command_name, func, command_config.opts)
   end
+
+  vim.api.nvim_create_user_command("Obsidian", function(data)
+    local command_name = data.fargs[1]
+    data.args = util.string_replace(data.args, command_name .. " ", "")
+    table.remove(data.fargs, 1)
+    M[command_name](client, data)
+  end, {
+    nargs = "+",
+    desc = "Run an obsidian.nvim command",
+    complete = function()
+      -- TODO: handle completion correctly for the remaining arguments of each subcommand
+      local command_list = {}
+      for command_name, _ in pairs(M.commands) do
+        local command_base_name = util.string_replace(util.to_snake_case(command_name), "obsidian_", "")
+        table.insert(command_list, command_base_name)
+      end
+      return command_list
+    end,
```

Comment:
TODO: need to figure this part out for the completing the args to each subcommand correctly. At the moment this will just keep suggesting command names.